processor/tailsamplingprocessor/config.go

```diff
@@ -66,8 +66,10 @@ type NumericAttributeCfg struct {
 type StringAttributeCfg struct {
 	// Tag that the filter is going to be matching against.
 	Key string `mapstructure:"key"`
-	// Values is the set of attribute values that if any is equal to the actual attribute value to be considered a match.
+	// Values is the set of attribute values or regexes that if any is matched the actual attribute value
 	Values []string `mapstructure:"values"`
+	// CacheSize is the maximum number of matched attribute values that is considered to be filtered
```

Comment:
Could you please clarify this? Just by reading this, I'm not sure I understand what it means.

Reply:
Updated. Thx.